datalad/cmdline/helpers.py

```diff
@@ -31,11 +32,25 @@ def __call__(self, parser, namespace, values, option_string=None):
             # lets use the manpage on mature systems ...
             try:
                 import subprocess
+                # extract version field from datalad manpage if present, swallow_error_logs
+                man_th = subprocess.check_output(
+                    ['zgrep', '\\.TH', '/usr/share/man/man1/datalad.1.gz'],
```

Comment:
ha ha ;-) you just used gzip! cheater! ;)  it might not be there e.g. on windows (eventually) or even on OSX.

Please use fileinput module, here is one liner:
```shell
$> python -c 'import fileinput; print(next(l for l in fileinput.input("/usr/share/man/man1/datalad.1.gz", openhook=fileinput.hook_compressed) if l.startswith(".TH")))'                                                      
.TH datalad 1 2016\-06\-27
```

and it shouldn't just open datalad.1.gz probably by corresponding manpage (see parser.prog.replace  call below -- store it into a variable)